flink-container/src/main/java/org/apache/flink/container/entrypoint/ClassPathJobGraphRetriever.java

```diff
@@ -63,8 +63,7 @@
 	@Nonnull
 	private final String[] programArguments;
 
-	@Nullable
-	private final String jobClassName;
+	private final Optional<String> jobClassName;
```

Comment:
These changes should be unrelated if I'm not mistaken. Please revert.

Reply:
Done. I originally did these changes, because I did not want to mix `@Nullable` xor `Optional` for fields.